docs/UserGuide.adoc

```diff
@@ -154,11 +154,18 @@ If you no longer need a note, or accidentally create a wrong one, don't worry. Y
 Format: `delete [-t] TITLE` +
 Example: `delete -t Notably/CS2103`
 
-.Demo for the `delete` command
-image:demo/Delete1.png[]
+[IMPORTANT]
+If your target path is a subtree, the whole subtree including the child would be deleted.
+
+.Demo for the delete command
+image::demo/Delete1.png[]
+
+.Demo for the delete command (after deletion)
+image::demo/Delete2.png[]
+
+[NOTE]
+You can achieve the same result by using a `AbsolutePath` instead. `delete -t /Workspace/Notably`
```

Comment:
There's a grammatical error here. It should be "an `AbsolutePath`".

What do you think about this rephrasing: "You can achieve the same result as above by using an `AbsolutePath` instead by executing `delete -t /Workspace/Notably`."?